extensions/amp-story-auto-ads/0.1/story-ad-page.js

```diff
@@ -127,6 +139,9 @@ export class StoryAdPage {
 
     /** @private {boolean} */
     this.viewed_ = false;
+
+    /** @private @const {!../../amp-story/0.1/amp-story-store-service.AmpStoryStoreService|!../../amp-story/1.0/amp-story-store-service.AmpStoryStoreService} */
```

Comment:
Is it desirable to have inter-version dependency?

Reply:
Good point! We should clean it but that'd be on the AMP Story WG to do it when we delete all the `0.1` code